src/kiota/KiotaHost.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using Kiota.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Kiota {
+    public static class KiotaHost {
+        public static RootCommand GetRootCommand()
+        {
+            var configuration = LoadDefaultConfiguration();
+            var outputOption = new Option("--output", "The ouput path of the folder the code will be generated in.") { Argument = new Argument<string>(() => "./output") };
+            outputOption.AddAlias("-o");
+            
+            var languageOption = new Option("--language", "The language to generate the code in.") { Argument = new Argument<GenerationLanguage?>(() => GenerationLanguage.CSharp) };
```

Comment:
`The language to generate the code in.` becomes
`The target language for the generated code files. Potential values are: csharp, java, or typescript.`

Reply:
The Enum validator already does that reflection work and lists out the available languages when someone passed something wrong :)